charts/spring-cloud-config-server/values.yaml

```diff
@@ -58,5 +58,5 @@ config:
   gitSearchpath: "{application}"
 
 secrets:
-  gitUsername:
-  gitPassword:
+  gitUsername: 
```

Comment:
you've deleted the quotes i've added to fix the linter...

Reply:
Sorry @monotek my bad